### ISSUE TITLE
Lag on iPad and OSX not loading properly after Enabling SIP

### ISSUE THREAD

**Issue by MehrabMirza** [pookjw/SidecarPatcher#70]
Hi All,
Have any of you noticed major lag on your ipad?
How did you fix that? 
Also, when I re-enable SIP. My OSX doesn't even load properly.
My Adobe Creative cloud goes haywire and Finder keeps quitting unexpectedly.

Here is what I'm running.
iPad Pro 10.5
iPad OS 13.5

Macbook Pro 13" Early 2015
MacOS Cataline 10.15.4
2.9GHz i5 | 16GB Ram 

Has anyone else experienced this? Can anyone help? I'm currently running everything with SIP disabled.
TIA

**Comment by anujsindgi** [comment]
Hi, I am experiencing the same thing after enabling SIP. I can't open Finder, Safari or even System Preferences. On top of it, I can't even see Bluetooth, WiFi, Clock on the top right corner. Is there any fix for this?

**Comment by 6mr9** [comment]
> Hi, I am experiencing the same thing after enabling SIP. I can't open Finder, Safari or even System Preferences. On top of it, I can't even see Bluetooth, WiFi, Clock on the top right corner. Is there any fix for this?

Yes there is fix thanks to apple. If you install new update which is newer than your version (which is macOS Catalina 10.15.6 on mine) the sidecar is gone but it makes the backup automatically after you update, enable SIP on terminal and there you go.
